doc/schema/schema.md

```diff
@@ -68,16 +68,18 @@ Samples from the posterior distribution p(theta|y).
 ### `sample_stats`
 Information and diagnostics for each `posterior` sample, provided by the inference backend. It may vary depending on the algorithm used by the backend (i.e. an affine invariant sampler has no energy associated). The name convention used for `sample_stats` variables is the following:
 * `lp`: (unnormalized) log probability for sample
-* `step_size`
-* `step_size_bar`
+* `step_size`: the step size used for this sample
+* `step_size_bar`: the current best known step-size
 * `tune`: boolean variable indicating if the sampler is tuning or sampling
-* `depth`:
-* `tree_size`:
-* `mean_tree_accept`:
+* `depth`: the depth of the tree that was used to generate this sample
+* `tree_size`: the number of leafs of the sampling tree, when the sample was accepted
+* `n_leapfrog`: number of steps selected in each iteration to traverse the posterior
+* `mean_tree_accept`: the mean acceptance probability for the tree that generated this sample
 * `diverging`: HMC-NUTS only, boolean variable indicating divergent transitions
-* `energy`: HMC-NUTS only
-* `energy_error`
-* `max_energy_error`
+* `energy`: HMC-NUTS only, the energy at the point in phase-space where the sample was accepted
+* `energy_error`: HMC-NUTS only, the difference in energy between the start and the end of the trajectory
+* `max_energy_error`: HMC-NUTS only, the maximum difference in energy along the whole trajectory
+* `accept/accept_stat`: acceptance probability for each sampler, when multiple samplers are used
```

Comment:
We should stick to a single name, is this because from_pymc uses one name and from_pystan another?

Reply:
Yes, it'll depend on the library used

Reply:
I think currently only real `sample_stats` parameter is `divergent`. Rest of them are still optional. I would not start renaming optional parameters until we use them for something.

Reply:
Yep, I also wanted to use this opportunity to add `max_tree_depth` (boolean) or something similar in order to be able to reproduce plots from https://arxiv.org/abs/1903.08008 (e.g. figures 3 or 4)

Reply:
Should ```depth``` and ```treedepth``` be renamed to ```max_tree_depth``` then?

Reply:
No, this would be a new variable, it would have to be boolean like `diverging`. One of the possible arguments to pass to NUTS is the maximum tree depth; many samples reaching this maximum depth can be an indicator of sampling or modeling issues.